src/legacy/core_plugins/metrics/public/components/vis_editor_visualization.js

```diff
@@ -128,19 +140,42 @@ class VisEditorVisualization extends Component {
       this.setState((prevState) => {
         const newHeight = prevState.height + (keyCode === keyCodes.UP ? -15 : 15);
         return {
-          height: Math.max(MIN_CHART_HEIGHT, newHeight)
+          height: Math.max(MIN_CHART_HEIGHT, newHeight),
         };
       });
     }
   }
 
+  hasShowPanelIntervalValue() {
+    const type = get(this.props, 'model.type', '');
+
+    return [
+      'metric',
+      'top_n',
+      'gauge',
+      'markdown'
+    ].includes(type);
+  }
+
+  getFormattedPanelInterval() {
+    const interval = get(this.props, 'model.interval', 0);
+
+    if (this.hasShowPanelIntervalValue() && interval) {
+      return interval === 'auto'
```

Comment:
This will send an interval of `>=1m` into the `convertStringIntervalIntoUnit` branch which will fail -> nothing is displayed in this case. We should either use this.state.panelInterval here or extend the regex in `get_interval` to also deal with this case (probably the first one)